install-app.sh

```
@@ -5,16 +5,16 @@
 # runs scripts under root user. 
 
 # install system updates and isntall python3-pip package using apt. '-yq' flags are 
-# used to suppress any interactive prompts - we won't be able to confirm operation 
-# when running the script as VM extention.  
+# used to suppress any interactive prompts - we won't be able to confirm operation
+# when running the script as VM extention.
```

Typo and also an opportunity for clarity. Fix spelling `extention` -> `extension`. Also move `set -euo pipefail` to the top of the script so the script fails fast when any command errors (important for VM extension executions).